### ISSUE TITLE
[wg/ag] Deque's concerns on the proposed AGWG charter for 2026 - 2028

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/charter-drafts#830]
The biggest problems AGWG faced with WCAG 3.0 when they started the '23 \- '25 charter were:

1. no clear scope (what's in, what's not)  
2. no timeline of what to publish when  
3. a lengthy list of ambitious goals without a viable solution.

We are encouraged by how much work AGWG has done on WCAG 3.0 the last charter. But regret that most of that time went to writing requirements rather than actually completing the deliverables of its charter. Addressing these problems is essential. WCAG 3.0 needs a realistic plan that has the support of its own group, the W3C and the wider digital accessibility community.

For its new charter AGWG has proposed to move ahead without such a plan. Its intent instead appears to be a time-boxed effort; Get as much done as possible before 2028 and punt on the rest, with at minimum an equivalent scope to WCAG 2.2. Even with this minimal scope we are concerned this timeline is far too aggressive, resulting in:

1. a lack of thorough review resulting in quality issues,  
2. addressing WCAG 2 gaps gets deprioritized in order to meet WCAG 3.0's MVP goal, and  
3. lengthy delays and objections to CR because of those two problems.

Deque urges AGWG to extend its current charter so that its deliverables can be completed: Build out a scope and plan for WCAG 3.0 with clear indications of what's in, what's at risk, and what's out. This must include a realistic timeline for what is delivered when, and includes releasing some parts of WCAG 3.0 work no later than 2028\. AGWG needs a real plan so that important work can be shipped faster, and so limited resources aren't spent on work that can wait. This plan must be approved by the working group, and must be part of the 2026 \- 2028 charter in order to get explicit support of it by W3C membership.

Deque’s goal is to guarantee that WCAG 3.0 is published as a REC with new requirements that fill known gaps \+ new conformance model prior to 2030\. Deque is very dissatisfied by the amount of time that has been spent on WCAG 3.0 with no positive impact on people with disabilities.

**Comment by iadawn** [comment]
Thank you for the feedback @WilcoFiers. I would urge you to provide a PR with the changes that you feel would address the problems you raise. This will assist in understanding the specifics of the concerns you perceive.

To clarify some specific points that you raise.

* For its new charter AGWG has proposed to move ahead without such a plan

	The proposed charter includes a link to the [WCAG 3 Project Plan](https://github.com/w3c/wcag3/wiki#WCAG3_Project_Plan). This links to information on the [publications and deliverables](https://github.com/w3c/wcag3/wiki/Publications-&-Deliverables) which seeks to expand upon what is included in the charter, and the [schedule](https://github.com/w3c/wcag3/wiki/Schedule) which provides an outline of the objectives and schedule for the upcoming work.

* Deque urges AGWG to extend its current charter so that its deliverables can be completed

	According to the [existing charter the deliverables](https://www.w3.org/2023/11/ag-charter#deliverables) include WCAG 3 working draft, ACT rules 1.1 as normative specifications and a range of informative materials and WCAG 2 maintenance. These have been completed.
  
  Additionally, we cannot extend the current charter anymore (at least beyond October 2026). The current charter has already been extended twice. We [endeavor not to extend charters beyond one year](https://www.w3.org/guide/process/charter.html#extension-request). There needs to be an AC review and a newly approved charter.

* At least some parts of WCAG 3.0 should become recommendation by the end of this charter

	Included in the scope of the new charter is the opportunity to [release interim publications](https://www.w3.org/2026/06/ag-wg-charter.html#wcag3-interim) when the material is mature. AGWG has explored how to create some sort of more modular approach to WCAG but there has been no viable approach demonstrated. The latest work on this was reported back in the [meeting of 3 October 2024](https://www.w3.org/2024/09/03-ag-minutes.html#t01).

A number of points raised cover the time line:

* This must include a realistic timeline for what is delivered when
* Deque’s goal is to guarantee that WCAG 3.0 is published as a REC
* Provide a realistic timeline for when different WCAG 3.0 documents will be released

Presupposing that the [existing plan](https://github.com/w3c/wcag3/wiki/Schedule) does not meet these concerns these do have two broad challenges. Predicting the future in a standardization group remains difficult because there are too many dimensions and parties involved for which we all have very limited control over. A reasonable plan has been set out and we will work towards meeting this. The second challenge is more about embedding detailed planning information within the charter. Project plans will be adjusted as more information and clarity around the project becomes apparent. This is a moving and changing document which is ill-suited to being fixed within a charter which will be set for a couple of years.

One possible approach to address concerns over timeline is to reduce the scope of the work. Limiting to anything less than WCAG 2.2, even with the improvements in clarity and approach, would be far from ideal. Fortunately, the work has already pushed beyond WCAG 2 in a number of areas. You highlight some gaps, plain language requirements, 3rd party content, scoring, XR, but these are all actively being explored.

As noted above, if there are specific changes that you feel would address your concerns, please raise a PR for consideration and discussion.

**Comment by murata2makoto** [comment]
I would like to add one additional perspective from the viewpoint of internationalization.

As part of the Text and Wording subgroup, I have been preparing a draft document entitled [Japanese Accessibility Requirements for the Web](https://docs.google.com/document/d/1-sd-fmvZYXpyUPCVKYpWz-UEjbYxyitBrACGNmAS374/edit?pli=1&tab=t.0#heading=h.346bh8atyzru).

This document is still under development, but one thing has already become very clear: language- and script-specific accessibility requirements extend far beyond text metrics or typography. The current draft includes requirements related to ruby, vertical writing, phrase boundaries (bunsetsu), page progression, kanji, and other characteristics of Japanese.

More importantly, I do not believe Japanese is unique.

It seems likely that similar documents could eventually be needed for Chinese, Arabic, Hindi, Thai, and many other languages and writing systems. Each of these may introduce accessibility requirements that are not yet reflected in WCAG.

If this assumption is correct, it raises an architectural question rather than merely a project management question.

A single, stable, monolithic specification may be difficult to evolve as additional language-specific requirements continue to be identified. Even if WCAG 3 reaches Recommendation status, future work on internationalization is unlikely to stop. New requirements will continue to emerge as more language communities systematically document their accessibility needs.

For this reason, I believe the Working Group should explicitly consider how language- and script-specific accessibility requirements can be incorporated and maintained over time. This may require an architecture that supports incremental evolution rather than assuming that all requirements can be collected into a single comprehensive specification.

I do not present "Japanese Accessibility Requirements for the Web" as a complete requirement document. Rather, I hope it serves as a concrete example of the kinds of requirements that may arise from one language community, and why the long-term architecture of WCAG deserves careful consideration from an internationalization perspective.

**Comment by iadawn** [comment]
@murata2makoto Note that part of the Text and Wording subgroup remit is to explore how to address requirements from different scripts and/or languages. This work started by looking at whether a Registry could be leveraged to address the issue that you outline. Unfortunately this was not found to be viable. An emerging idea is to explore script/language  features related to accessibility and create requirements for them. This might allow for more script/language support with less complication in how WCAG3 is managed. As noted the place to explore this is in the Text and Wording subgroup in the first instance.

As has been noted in a number of communications  modularising WCAG would undermine uses in regulations and legislation and would create problems given the overlapping nature of accessibility considerations which would fefy strict categories. This was most recently highlighted in an email to you Murata-san from Alastair on 2026-7-8.
